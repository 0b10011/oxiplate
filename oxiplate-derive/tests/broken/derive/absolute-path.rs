use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate = "/absolute"]
struct Data;

fn main() {
    print!("{}", Data);
}