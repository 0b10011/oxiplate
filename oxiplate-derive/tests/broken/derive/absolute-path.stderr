error: proc-macro derive panicked
 --> tests/broken/derive/absolute-path.rs:3:10
  |
3 | #[derive(Oxiplate)]
  |          ^^^^^^^^
  |
  = help: message: Template path could not be canonicalized; Provided: /absolute

error[E0277]: `Data` doesn't implement `std::fmt::Display`
 --> tests/broken/derive/absolute-path.rs:8:18
  |
8 |     print!("{}", Data);
  |             --   ^^^^ `Data` cannot be formatted with the default formatter
  |             |
  |             required by this formatting parameter
  |
  = help: the trait `std::fmt::Display` is not implemented for `Data`
  = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
  = note: this error originates in the macro `$crate::format_args` which comes from the expansion of the macro `print` (in Nightly builds, run with -Z macro-backtrace for more info)
